packages/plugin-react/src/fs.ts

```diff
@@ -0,0 +1,12 @@
+import fs from 'fs'
+
+export function readJsonFile(file: string, throwError?: boolean): any {
```

Comment:
We could improve this with a template so we can assume a return type

```suggestion
export function readJsonFile<T = any>(file: string, throwError?: boolean): T {
```